website/tutorial/basic/10-subscriptions.mdx

```diff
@@ -0,0 +1,617 @@
+---
+id: 09-subscriptions
+title: Subscriptions
+sidebar_label: Subscriptions
+---
+
+In this section, you'll learn how you can bring realtime functionality into your app by implementing GraphQL subscriptions. The goal is to implement two subscriptions to be exposed by your GraphQL server:
+
+- Send realtime updates to subscribed clients when a new `Link` element is _created_
+- Send realtime updates to subscribed clients when an existing `Link` element is _upvoted_
+
+### What are GraphQL subscriptions?
+
+Subscriptions are a GraphQL feature that allows a server to stream data to its clients when a specific _event_ happens. Subscriptions are just part of your GraphQL contract, and they refer to _events_. To be able to send these _events_ in real-time, you need to choose a transport that has support for that.
+
+In this chapter of the tutorial, you are about to add GraphQL Subscriptions to your server, based on a transport called SSE (Server-Sent Events). This protocol is an extension of simple HTTP, with streaming and real-time capabilities, and doesn't require any special setup or a new server (as described before, there are many options to implement subscriptions, like WebSockets).
+
+Sever-Sent Events are a way to "upgrade" a basic HTTP request into a long-living request that will emit multiple data items. This is a perfect fit for GraphQL Subscriptions, and implementing and scaling is just simpler than WebSockets.
+
+### Implementing GraphQL subscriptions
+
+You'll be using a simple `PubSub` implementation from the `@graphql-yoga/node` library to implement subscriptions to the following _events_:
+
+- A new model is **created**
+- An existing model **updated**
+- An existing model is **deleted**
+
+> `Pub/Sub` refers to a technique used to create a messaging pattern, where some parts of the code publishes events/messages, and other parts of the code subscribes and being notified about the events/messages. You are going to use that technique in order to create a simple subscription for the GraphQL Subscriptions, based on events published by the GraphQL mutations.
+
+You will do this by first adding an instance of `PubSub` to the context, just as you did with `PrismaClient`, and then calling its methods in the resolvers that handle each of the above events.
+
+### Setting up our `PubSub` instance
```

Comment:
an illustration would be beneficial here to show the need for PubSub "architecture"

Reply:
noted here: https://github.com/dotansimha/graphql-yoga/issues/916